src/ripple/basics/StringUtilities.h

```diff
@@ -148,6 +148,15 @@ trim_whitespace(std::string str);
 boost::optional<std::uint64_t>
 to_uint64(std::string const& s);
 
+/** Determines if the given string looks like a valid TOML-file hosting domain.
+
+    Do not use this function to determine if a particular string is a valid
+    domain, as this function may reject domains that are otherwise valid and
+    doesn't check whether the TLD is valid.
+ */
+bool
+isValidTomlDomain(std::string const& domain);
```

Comment:
The comment and the function name seem to be at odds with one another. Maybe a name like `isPossibly...` or `isSeemingly...` might communicate the purpose of the function more clearly. Up to you.

Reply:
You're right. I'll improve the naming and/or comment.